docker/crontab

```diff
@@ -1,8 +1,5 @@
-PATH=/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin
-PYTHONPATH=/usr/local/lib/python27.zip:/usr/local/lib/python2.7:/usr/local/lib/python2.7/lib-dynload:/usr/local/lib/python2.7/site-packages
-
 # Compute stats
-5 * * * * /usr/local/bin/python /code/manage.py compute_stats >> /var/log/stats_calc.log 2>&1
+5 * * * * root /usr/local/bin/python /code/manage.py compute_stats >> /var/log/stats_calc.log 2>&1
```

Comment:
Is there a reason to run this as root ?
Also what's this log file ? How is it rotated / size-controlled ?

Reply:
No strong reason to be root other than "it's easy". I'm happy to change this to any other user. The easiest would be www-data, like the webserver, as this allows us to share permissions. The next easiest would be to have a single acousticbrainz user in this image which is shared over all containers. Because this will potentially write to volumes mounted from a host, we should consider having a metabrainz standard of UIDs for each service that we can make consistent on all hosts.

This logfile will contain the output of the script that's run in the cronjob. Is there any logrotate happening in other MeB images that I can copy? The alternative is to redirect to the container's stdout so that we let docker deal with the logs

Reply:
Logging to container stdout/stderr now.